requirements.txt

```diff
@@ -28,13 +26,12 @@ py-ed25519-bindings~=0.1.2
 py-bip39-bindings>=0.1.6
 pytest>=6.1.2
 pytest-xdist==2.2.0
-pytest-asyncio>=0.14.0
 pytorch_transformers
 pyyaml
 rich
 rollbar
-requests~=2.25.0
-scalecodec~=0.10.59
+requests>=2.25.0
+scalecodec~=0.11.16
```

Comment:
do we need this at all?

Reply:
Yes we apparently do, it broke my shit before. Not sure if that was me or not :/